engine-tests/src/test/java/org/terasology/engine/core/module/ClasspathCompromisingModuleFactoryTest.java

```diff
@@ -15,17 +15,29 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
+import static com.google.common.truth.Truth8.assertThat;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class ClasspathCompromisingModuleFactoryTest {
     static final Class<?> someClassOutsideTheModule = ClasspathCompromisingModuleFactory.class;
+    static final String METADATA_NAME = "module.json";
```

Comment:
Do you use a different metadata file name here on purpose from what we use in Terasology? I think we're going with `module.txt`, even though I'd like to use `module.json` for my editors to understand the file format ^^

Reply:
The trade-off here is not wanting to spend too many lines of code in setUp overriding defaults that aren't significant to the test cases.

and the default name isn't exposed as a constant by gestalt.

but I do agree that the test cases shouldn't look too different than the values we see in practice, which is why I did set the libs path.

so yeah, I'm wishy-washy on this one and open to making changes, just having a hard time settling on what makes things most clear.

Reply:
I think the test cases are clear enough, no need to change anything. I was just wondering whether there's a deeper reason for this 🙃